azure-webapp-maven-plugin/README.md

```diff
@@ -1,186 +1,179 @@
-                       </deployment>                        
-                   </configuration>
-               </plugin>
-           </plugins>
-       </build>
-    </project> 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.microsoft.azure</groupId>
+                <artifactId>azure-webapp-maven-plugin</artifactId>
+                <!-- check Maven Central for the latest version -->
+                <version>1.5.3</version>
+                <configuration>
+                    <schemaVersion>v2</schemaVersion>
+                    <resourceGroup>${RESOURCEGROUP_NAME}</resourceGroup>
+                    <appName>${WEBAPP_NAME}</appName>
+                    <runtime>
+                        <os>linux</os>
+                        <javaVersion>jre8</javaVersion>
+                        <webContainer>tomcat 8.5</webContainer>
+                    </runtime>
+                    <deployment>
+                        <resources>
+                            <resource>
+                                <directory>${project.basedir}/target</directory>
+                                <includes>
+                                    <include>*.war</include>
+                                </includes>
+                            </resource>
+                        </resources>
+                    </deployment>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>      
     ```
-   
-3. Use the following commands to deploy your project to Azure App Service. 
-
+3. Use the following commands to deploy your project to Azure App Service.
     ```
     $ mvn azure-webapp:deploy
     ```
 
-<a name="samples"></a>
-## Samples	
-A few typical usages of Maven Plugin for Azure App Service Web Apps are listed at [Web App Samples](../docs/web-app-samples.md). You can choose one to quickly get started.
-
 <a name="goals"></a>
 ## Goals
 
-The Maven Plugin for Azure App Service Web Apps has only one goal: `azure-webapp:deploy`. 
-
-Goal | Description
---- | ---
-`azure-webapp:deploy` | Deploy artifacts or docker container image to an Azure Web App based on your configuration.<br>If the specified Web App does not exist, it will be created.
-## Authentication with Azure
-
-You can use the Azure CLI 2.0 for authentication. More authentication methods can be found [here](../docs/common-configuration.md).  
-
-1. Install the Azure CLI 2.0 by following the instructions in the [Install Azure CLI 2.0](https://docs.microsoft.com/cli/azure/install-azure-cli) article.
-
-2. Run the following commands to log into your Azure subscription:
-
-    ```
-    $ az login
-    $ az account set --subscription <subscription Id>
+#### `azure-webapp:config`
+-  A plugin goal providing a great experience for managing the configuration of [azure-webapp:deploy](#azure-webappdeploy), it will rewrite your pom configuration based on your input. Details about the configuration could be found in the following.
```

Comment:
it will add configurations for your App Service into your pom.xml or rewrite existing ones based on your input.